build.gradle

```diff
@@ -158,14 +158,18 @@ tasks.register('testNaming') {
 	}
 }
 
+enum Modifiers {
+	DevMode, GenDocs, Debug, Profile
```

Comment:
This is not the correct casing for enums.
You may use either the typical JVM `UPPER_SNAKE_CASE` or follow groovy-lang's example of `lowercase`.